doc/moveit_grasps/moveit_grasps_tutorial.rst

```diff
@@ -140,7 +140,16 @@ The result should look something like this:
 
 3) Visualize Grasp Generation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-To visualize grasp generation for randomly placed blocks::
+
+This tool demonstrates several concepts around how scoring works during grasp generation.
+It bases the chosen grasp using the following approaches:
+
+ - *Ideal TCP Grasp Pose*: the generator will bias the chosen grasp based on an input of preferred grasp. This allows you to say, for example, 'I want it to be as close as possible to a grasp that is dead-ceneted on the object'.
+ - *Pose-Based Grasp Score Weighting*: Bias certain translation or rotation axes to score higher, by increasing the weight of that specific axis.
```

Comment:
nit : This is used for more than translation and rotation. It is also used for grasp depth, finger width, and suction gripper overhang.